worldedit-core/src/main/java/com/sk89q/worldedit/extent/clipboard/io/share/BuiltInClipboardShareDestinations.java

```diff
@@ -75,7 +75,7 @@ public Consumer<Actor> share(ClipboardShareMetadata metadata, ShareOutputProvide
 
         @Override
         public ClipboardFormat getDefaultFormat() {
-            return BuiltInClipboardFormat.SPONGE_SCHEMATIC;
+            return BuiltInClipboardFormat.SPONGE_V2_SCHEMATIC;
```

Comment:
I guess given upstream defaults to this then we ought to

Reply:
ah, might be a good idea to use the fast impl there to - no need to use the upstream implementation when the schematic is conform either way.

Reply:
I now see that this is for the EngineHub Paster - which I would not want to touch. I'll update the supported + default format for Ark though,